compiler/rustc_ast/src/ast.rs

```diff
@@ -1478,10 +1478,10 @@ impl MacArgs {
     }
 
     pub fn span(&self) -> Option<Span> {
-        match *self {
+        match self {
             MacArgs::Empty => None,
             MacArgs::Delimited(dspan, ..) => Some(dspan.entire()),
-            MacArgs::Eq(eq_span, ref tokens) => Some(eq_span.to(tokens.span().unwrap_or(eq_span))),
+            MacArgs::Eq(_, token) => Some(token.span),
```

Comment:
Is it intentional that we're no longer including `eq_span`?

Reply:
No, this is a mistake, will fix.